apps/backend/__tests__/login.test.ts

```
@@ -0,0 +1,116 @@
+
+const prisma = new PrismaClient();
+
+function createMockRequest(body: { email: string; password: string }) {
+    return {
+        json: async () => body,
+    };
+}
+
+describe("POST Login Endpoint", () => {
+    beforeAll(() => {
+        process.env.JWT_SECRET = process.env.JWT_SECRET;
```

Biome (1.9.4)

[error] JWT_SECRET is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)